process/archiving.md

```diff
@@ -0,0 +1,35 @@
+# CNCF Project Archiving Process v1.0
+
+Open source projects have a lifecycle and there are times that projects become inactive due to a variety of reasons. There are also cases where a project may no longer want to be supported by the TOC.
+
+## Archiving Criteria
+
+There are different criteria to consider when archiving a project, but here are ones that the TOC looks for:
+
+* It hasn't seen a commit in over 6 months.
+* It hasn't seen a release in over 6 months.
+* There haven't been any issues opened for 6 months.
+* Opened issues haven't received a response within 6 months.
+* It's binaries/source are no longer being downloaded
```

Comment:
Given the number of companies relying on CNCF projects, I think 3 months may be a better milestone here.

Reply:
Assuming a 3 month release cycle (on average), it probably makes sense to have folks miss 2 releases (6 months) i feel

Reply:
I’m not sure that having timelines is terribly helpful - they could easily be gamed to keep a project open. I’d think we could just leave it at the discretion of the TOC to say that a project no longer meets the criteria for inclusion.

Reply:
@lizrice SGTM.  But also worth setting approximate expectations.  6 months seems fine to me.